GermanFilter/sections/specific.txt

```diff
@@ -67,6 +67,8 @@ haz.de,ostsee-zeitung.de,sn-online.de,dnn.de,kn-online.de,ln-online.de,neuepress
 !
 radiosaw.de###oms_gpt_rectanglesidebar
 fitbook.de##a[href^="https://secure.adnxs.com/"]
+||hasepost.de/wp-content/banners/
```

Comment:
||hasepost.de/wp-content/banners^

Reply:
If I'm not wrong both rules work the same. Please explain why we need `^` in the rule?

Reply:
`^` is unnecessary here. It is useful if url contains separators 
https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters#special-characters